app/src/main/java/com/kickstarter/ui/activities/ProjectActivity.kt

```diff
@@ -484,11 +484,11 @@ class ProjectActivity : BaseActivity<ProjectViewModel.ViewModel>(), CancelPledge
         return pledgeReason == PledgeReason.PLEDGE || pledgeReason == PledgeReason.UPDATE_REWARD
     }
 
-    private fun renderProject(projectAndNativeCheckoutEnabled: Pair<Project, Boolean>) {
-        val project = projectAndNativeCheckoutEnabled.first
-        val nativeCheckoutEnabled = projectAndNativeCheckoutEnabled.second
-        this.adapter.takeProject(project, nativeCheckoutEnabled)
-        ProjectViewUtils.setActionButton(project, this.back_project_button, this.manage_pledge_button, this.view_pledge_button)
+    private fun renderProject(projectDataAndNativeCheckoutEnabled: Pair<ProjectData, Boolean>) {
+        val projectData = projectDataAndNativeCheckoutEnabled.first
+        val nativeCheckoutEnabled = projectDataAndNativeCheckoutEnabled.second
```

Comment:
Is this still feature-flagged on Android? We've removed old checkout completely on iOS.

Reply:
Yes 😢 but I have a ticket in JIRA in the backlog to remove it!